client/my-sites/term-tree-selector/docs/example.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import TermTreeSelector from '../';
+import observe from 'lib/mixins/data-observe';
+import sites from 'lib/sites-list';
+
+const TermTreeSelectorExample = React.createClass( {
+	mixins: [ observe( 'sites' ), PureRenderMixin ],
+
+	render() {
+		const primary = this.props.sites.getPrimary();
+
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
The precedent is to use `docs__design-assets-group` (which applies the same styles):

https://github.com/Automattic/wp-calypso/blob/master/client/components/ellipsis-menu/docs/example.jsx

Though I'm a bit confused in the above example why it's not warning for being `docs`, since it's the default export (not true here).